db/schema.rb

```diff
@@ -33,11 +33,15 @@
     t.string "magical_effects"
     t.decimal "unit_weight", precision: 5, scale: 2, null: false
     t.boolean "dragon_priest_mask", default: false
-    t.boolean "quest_item", default: false
-    t.boolean "unique_item", default: false
     t.boolean "enchantable", default: true
+    t.boolean "leveled", default: false
```

Comment:
I have no idea why these fields weren't in the schema before because they were definitely in the  migrations - and were additionally somehow present in the production database (but not my dev environment, even after I dropped and re-setup the database).